.env

```diff
@@ -14,7 +14,7 @@ ACP_BASE_URL=authorization.cloudentity.com:8443
 ACP_MTLS_URL=https://authorization.cloudentity.com:8443
 
 # acp version
-ACP_VERSION=2.0.0
+ACP_VERSION=03cee8d
```

Comment:
Can we tag this image under some name and push it, since this commit hash image will probably be removed some time soon?

Reply:
Good point, but I talked with @ikawalec about it and there is another ACP fix pending so we can tag it then